source/application/SXRMB/SXRMBAppController.cpp

```diff
@@ -374,7 +374,6 @@ void SXRMBAppController::setupUserInterface()
 	CLSCrossHairGeneratorControlView *crossHairView = new CLSCrossHairGeneratorControlView(SXRMBBeamline::sxrmb()->crossHairGenerator());
 	SXRMBCrystalChangeView *crystalChangeView = new SXRMBCrystalChangeView(SXRMBBeamline::sxrmb()->crystalSelection());
 	CLSJJSlitsView *jjSlitsView = new CLSJJSlitsView(SXRMBBeamline::sxrmb()->jjSlits());
-	jjSlitsView->setDataRange(18, 0);
```

Comment:
The dataRange is added for SXRMB because of a bug. I'm not sure removing this line will cause the bug back or not.